vignettes/tidytranscriptomics.Rmd

```diff
@@ -761,6 +762,111 @@ de_gate %>%
   select(edgerQLT_logFC, deseq2_log2FoldChange)
 ```
 
+## Study of cellularity
+
+For this example we will use a subset of the breast cancer TCGA dataset. 
```

Comment:
I think it would be good to add a sentence or two here for why a researcher might want to do this, something like... "If we are sequencing tissue samples, we may want to know what cell types are present, and if there are differences in expression between them. tidybulk has a `deconvolve_cellularity` function that can help us do this.

Reply:
I'll do